src/applications/mhv/landing-page/utilities/index.js

```diff
@@ -0,0 +1,16 @@
+function getDestinationDomain(href) {
+  const url = new URL(href, document.location.href);
+
+  const { hostname } = url;
+
+  if (hostname.includes('vagov')) {
+    return 'va.gov';
+  }
+  if (hostname.includes('mhvnp')) {
+    return 'mhvnp';
+  }
+
+  return hostname;
```

Comment:
To clarify what I'm thinking: I _think_ recording the actual hostname is preferable. I could be wrong, but I think something like the following would be good

```suggestion
  try {
  const { hostname } = new URL(href, document.location.href);
  } catch (e) {
  // handle error if URL constructor throws an error, e.g href is undefined
  return ""; // I don't actually know what we'd want to return here
  }

  return hostname;
```